control-plane/csi-driver/src/bin/node/dev/nvmf.rs

```diff
@@ -32,6 +33,7 @@ lazy_static::lazy_static! {
 pub(super) struct NvmfAttach {
     host: String,
     port: u16,
+    xprt: String,
```

Comment:
Can we not use the TrType?

Reply:
We could, however the existing structures e.g. `Subsystem` have transport as String which makes comparisons easier. TrType is only used on connect args builder side.